ftplugin/racket.vim

```diff
@@ -30,7 +30,7 @@ setl lispwords+=match-letrec,match-define,match-define-values
 setl lisp
 
 " Enable auto begin new comment line when continuing from an old comment line
-setl comments+=:;
+setl comments+=:;;;;,:;;;,:;;,:;
```

Comment:
Why use `+=` instead of `=`?

Vim's default `'comments'` is `s1:/*,mb:*,ex:*/,://,b:#,:%,:XCOMM,n:>,fb:-`.

Racket comments don't start with `//` or `%` ...